pt_br.lang

```diff
@@ -1,5 +1,3 @@
-Translator = Diego R. V. (mine.ly/DaDodger)
```

Comment:
We should probably keep this! Renaming the `Translator` key to something else would be a good idea though.

Reply:
This line is only in the Brazilian Portuguese translation file, and the JSON variant doesn't have it anyway.

You already have all translators listed on all your mod pages on Planet Minecraft. They are kept up to date. IMHO this plus Git commit history is enough. But as an enhancement you can, for example, make on your own website something similar to what henkelmax did, see: https://modrepo.de/minecraft/gravestone/credits

Reply:
It just feels weird to remove the credit that the author clearly wanted to be there. Not sure why exactly it doesn't transfer to the json variant, but if you change it to something like `xaero_pt_br_translator=Diego R. V. (mine.ly/DaDodger)` it probably will transfer.

Reply:
And I do have my own website, but it's definitely in desperate need of rework lol.

Reply:
If I remember correctly, DaDodger is an official proofreader of the Brazilian Portuguese translation for vanilla Minecraft. I can try to reach out to him and ask him what he thinks about what I'm doing here.

Reply:
I managed to contact him and got an answer. He said that I could remove this line. If you would like to see the full answer as a proof, I may PM you a screenshot somewhere.

Reply:
I would honestly prefer to just leave it there. I also messaged them the same way we originally got in contact and they are not responding. Where did you message them?

Reply:
I did it via Crowdin, where the official translations for vanilla Minecraft take place, and he is a proofreader there.

Reply:
Alright. I just checked and it seems to be really them. I guess we'll remove the line then.